applications/obsloctap/values.yaml

```diff
@@ -1,16 +1,37 @@
-image:
-  # -- obsloctap image to use
-  repository: "ghcr.io/lsst-dm/obsloctap"
+obsloctap:
+  image:
+    # -- obsloctap image to use
+    repository: "ghcr.io/lsst-dm/obsloctap"
 
-  # -- Pull policy for the obsloctap image
-  pullPolicy: "IfNotPresent"
+    # -- Pull policy for the obsloctap image
+    pullPolicy: "IfNotPresent"
 
-  # -- Tag of obsloctap image to use
-  # @default -- The appVersion of the chart
-  tag: "tickets-dm-38824"
+    # -- Tag of image to use
+    # @default -- The appVersion of the chart
+    tag: "tickets-DM-44729"
+  logLevel: verbose
+
+consumekafka:
+  image:
+    # -- obsloctap image to use
+    repository: "ghcr.io/lsst-dm/consumekafka"
+
+    # -- Pull policy for the obsloctap image
+    pullPolicy: "Always"
+
+    # -- Tag of image to use
+    # @default -- The appVersion of the chart
+    tag: "tickets-DM-44729"
+  logLevel: verbose
+
+rubinsim:
+  # for rubin sim
+  dataDir: "/sdf/data/rubin/shared/rubin_sim_data"
+  bucketVal: "1"
 
 # -- Environment variables (e.g. butler configuration/auth parms) for panel
-environment: {}
+environment:
```

Comment:
I didn't see this being used, maybe you should remove it?

Reply:
something int he phalanx build will not allow it without the {} -- i did not put it there ..